### ISSUE TITLE
The `on` prefix is not localized

### ISSUE THREAD

**Issue by yardenshoham** [github/relative-time-element#255]
Everything apart from this prefix is localized

**Comment by keithamus** [comment]
This is correct. The prefix is soft deprecated because of this. The correct solution will likely involve localising the surrounding text using other techniques that this element won’t deliver.

**Comment by yardenshoham** [comment]
Gotcha. In https://github.com/go-gitea/gitea/pull/23988 we did just that, localized `on`.

**Comment by silverwind** [comment]
It seems it's not possible to localize the "on" alone in [some languages](https://github.com/go-gitea/gitea/issues/24074), so a `on <date>` phrase needs to be localized as a whole.
